integration/flink/src/test/java/io/openlineage/flink/client/FlinkConfigParserTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+/* Copyright 2018-2023 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.flink.client;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import io.openlineage.client.OpenLineageYaml;
+import io.openlineage.client.transports.HttpConfig;
+import java.net.URI;
+import java.util.Optional;
+import lombok.SneakyThrows;
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ConfigOptions;
+import org.apache.flink.configuration.Configuration;
+import org.junit.jupiter.api.Test;
+
+public class FlinkConfigParserTest {
```

Comment:
Would be nice to check what happens when

- there are useless `openlineage.*` config options set
- when there are none required openlineage options

Reply:
Useless config options cause failure which is same behaviour as for Spark. This makes sense as if a user puts an incorrect config entry, it's better we throw an error than proceed silently. 

What do you mean by `none required openlineage options` ?